src/bindings/python/flux/job/list.py

```diff
@@ -97,6 +97,44 @@ def job_list_id(flux_handle, jobid, attrs=[]):
     return rpc
 
 
+# get_job is the single variant of job_list_id, and returns the
+# expected data structure (dict) to describe one job (or None)
+def get_job(flux_handle, jobid):
+    """Get job information dictionary based on a jobid
+
+    This is a courtesy function for the user that looks for it
```

Comment:
Suggestion: "This is a courtesy, blocking function for users looking for details about a job after submission."